search/query/boolean.go

```diff
@@ -61,41 +60,43 @@ func NewBooleanQueryMinShould(must []Query, should []Query, mustNot []Query, min
 
 // SetMinShould requires that at least minShould of the
 // should Queries must be satisfied.
-func (q *booleanQuery) SetMinShould(minShould float64) {
-	q.Should.(*disjunctionQuery).SetMin(minShould)
+func (q *BooleanQuery) SetMinShould(minShould float64) {
+	q.Should.(*DisjunctionQuery).SetMin(minShould)
 }
 
-func (q *booleanQuery) AddMust(m Query) {
+func (q *BooleanQuery) AddMust(m ...Query) {
```

Comment:
Yeah this was introduced to address the opposite case.  Previously, they have []bleve.Query{}, now that has to be come []query.Query{} thus necessessitating the import of the "query" package, which I assume many would rather avoid.  By writing it this way you can pass (q1, q2, q3) without needing to create the array first.

But your point is valid, just not sure where to document/demonstrate that.